### PR TITLE
Adds new chemicals, increases potency of several old ones

### PR DESCRIPTION
**Pull request by ghost**
...by making people hate it more because it's a lot stronger.

Adds a few new chemicals, ports hippiestation's rotatium.

The new chemicals added are-

replitium, a really weak poison...that replicates indefinitely in the victim's bloodstream over and over unless purged with calomel, meaning that either you take a fuckton of antitox or a little calomel, or over the course of 15 minutes you puke your asshole out and die.

hemoharragium, a chemical that tears away at the victim's bloodvessels, making them bleed internally and die. 

Maloculin, a chemical that causes blindness the longer it's in you.

Floroantimonic acid, a hilariously overpowered superacid that I have no way how I want to implement but is still fun to have exist if an admin is bored or something, that eats right through everything, eats away your internal organs, or boils away your blood. If I wanted to make it realistic, it'd actually EXPLODE on contact with your blood-and melt away your limbs-but I had no idea on how to implement that without making it spam admins with EXPLOSION AT X until the guy gibbed.

A few shitty traitor poisons that took too long to have their effects as compared to most chems which are instant stuns have been buffed to be very fast.

Chloral is no longer hilariously sad as a poison, doing light stamina damage to slow down the victim and knocking them out much faster.

A lot of poisons are a lot meaner.

Heparin is now actually realistic, seeing as it's actually fucking a medicine that's used to treat myocardial infarction and yet the dumbass who made it didn't bother to look into it. I would make it REALISTIC realistic, but honestly I have no idea how i'd want to code in it's actual effects and such, nor is our blood system good enough to do what I want. 

Slime jelly is less RNG lethal and more always lethal. Instead of randomly picking sometimes YOU DIE SUDDENLY, it now chooses between AUGH and FUCK HELP.

Plasma is better at igniting people if splashed on them, compared to welding fuel.

Lexorin is meaner, for such a sad joke chem that could be countered with the shit in your backpack.

Zombie powder is still an instant stun...for a short period of time. After that time passes ,no matter how much is in your bloodstream, it does absolutely nothing until the last bit is removed. So now, having it in the chem dispenser is okay, since it'll keep you unconcious instantly...but if the guy takes too much time no amount of zombie powder will save his ass since they won't fall unconcious until it's all out of their bloodstream.

Mindbreaker causes brain damage as well. (This will be much worse when/if the med system I proposed gets done.)

Sporetoxin and burning spore toxin are now nasty, as compared to the absolute jokes they were before. Yes, weavers DO inject spore toxin on low health, but don't forget they also drop antitox bottles, so you can patch yourself up very easily, and honestly sporetoxin isn't much worse than toxin itself.

Cyanide is no longer sad. It has shorter RNG stuns, but is far deadlier.

Sulfonal is mean.

Acid inflicts burn damage when injected, not brute. What moron did that?

Mint toxin now gibs fat people again. 

Chem neuro is now OP again.
#### Changelog

:cl:
rscadd: Chemistry has recieved several buffs. Experiment and find out.
rscadd: Several new chemicals have been added, albiet currently unavailable. 
rscadd: Tiger Co. has made their specific poisons far deadlier and more effective.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
i fucking hate you travis i hope you choke on 999999999999999999999999999999999999 zillion dicks

**Comment by Time-Green** [comment]
Mint gibs fat people, I am pretty sure I did that at one point.

**Comment by Time-Green** [comment]
Also fix 120 and 123.
You added a ) too much, it's pissing of travis

**Comment by oisin100** [comment]
He meant the first ), not the second one.

**Comment by BOBsHERE** [comment]
Is this real life?

**Comment by Kmc2000** [comment]
So, if i understand correctly, you want to give the station's chemists the ability to give everyone ebol-aids?

Yes...?
:+1:

**Comment by X-TheDark** [comment]
I'll take a better look at Floroantimonic Acid, because even I'm stumped by how it's organized.

**Comment by X-TheDark** [comment]
What's the difference between Heparin and Hemoharragium? They both basically do the same thing (remove blood), but Heparin does it better. I guess Hemoharragium is more stealthy, but I don't see why we need 2 chems that basically do the same thing. Nice touch on the heart attack, though.

Rotatium you can forget about, it bugs the client way, WAY too badly to be added (at least using the naive client.dir = X approach), I've done this when I added Heparin/that other drug purging chem in.

**Comment by X-TheDark** [comment]
I'll have to look at 'Floroantimonic Acid' some other time (I'll try doing it soon-ish) to reorganize it in a fashion that would not cause issues, isn't organized so badly and works as it's intended by Adam.

**Comment by ghost** [comment]
Heparin causes excessive bleeding, but does not directly cause bloodloss itself. If bandaged and countered properly it can be handled, and it also won't do much if there's no brute damage to start.

Hemoharragium actually eliminates the blood itself, basically being internal vs external bleeding-bandaging does nothing, and simply spamming bruisepacks won't help like ti does with Heparin.

Also could be useful to have in hemo rounds in snipers...

**Comment by X-TheDark** [comment]
I see, that's cool. Didn't know bandaging stopped Heparin.

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -113,24 +115,25 @@
...
-	if(prob(10))
+	if(prob(50))
 		M << "<span class='danger'>Your insides are burning!</span>"
-		M.adjustToxLoss(rand(20,60)*REM, 0)
+		M.adjustToxLoss(50*REM, 0)
 		. = 1
-	else if(prob(40))
-		M.heal_organ_damage(5*REM,0, 0)
+	else if(prob(50))
+		M.adjustCloneLoss(50*REM, 0)
```
Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm:123]: why? it already does stupidly high toxin damage

Comment (ghost) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm:123]: > random chance to heal you

y

Comment (X-TheDark) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm:123]: `adjustXLoss(positive_number)` causes DAMAGE and NOT HEALING (you'd need it to be a negative number for it to heal).

**Comment by ParagonPrime** [comment]
hemorrghagium in hemo rounds would be cool as hell, full support

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 1 above)
Comment (ShadowDeath6) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm:123]: X he meant that it healed limb damage randomly. 

This number is way too high, though.

**Comment by ShadowDeath6** [comment]
why did you make chloral a stronger knockout chem than zombie powder by the way

**Comment by ghost** [comment]
fucking i cant see what is breaking

**Comment by ghost** [comment]
@X-TheDark WAT IS BREAK

**Comment by Cruix** [comment]
You have some vars that are defined in different procs, so you'll have to add instance vars to keep track of that like I did with beepsky smash. I'll see if I can fix it this afternoon if the editor does not crash on me.

**Comment by X-TheDark** [comment]
@AdamElTablawy 
The things that are breaking are as follows:

Reagent by itself doesn't have a `method` variable. I assume you want this to happen when the stuff is injected/ingested (instead of every tick it's in your body), I was going to move this to `reaction_mob` (which runs when chem is introduced).

**Comment by Super3222** [comment]
If we are considering to merge this, than please by all means testmerge it. However, I suggest /NOT/ merging it directly. But somehow figuring out to upload the version with these changes or however tg does it. So we don't have to go through the trouble of reverting it, than having to pick through the pieces.

**Pull request closed without merging**